pkg/hwinfo/report.go

```diff
@@ -456,6 +459,7 @@ func (h HardwareDevice) MarshalJSON() ([]byte, error) {
 			SysfsId           string        `json:"sysfs_id,omitempty"`
 			SysfsBusId        string        `json:"sysfs_bus_id,omitempty"`
 			SysfsDeviceLink   string        `json:"sysfs_device_link,omitempty"`
+			SysfsIOMMUGroupId string        `json:"sysfs_iommu_group_id,omitempty"`
```

Comment:
We don't need the id here.
```suggestion
			SysfsIOMMUGroup int        `json:"sysfs_iommu_group,omitempty"`
```